WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModelTests.swift

```diff
@@ -4,7 +4,8 @@ import TestKit
 @testable import WooCommerce
 
 /// Temporarily removed pending a rewrite for the new InPersonPaymentsMenuViewModel #11168
-class InPersonPaymentsMenuViewModelTests: XCTestCase {
+@MainActor
```

Comment:
Question: If we're marking the whole `InPersonPaymentsMenuViewModelTests` as `@MainActor` for simplicity, should this change drive the implementation of `InPersonPaymentsMenuViewModel` and make it `MainActor` as well? (and final!). We seem to mark just a couple of methods with it.